### PR TITLE
Add Contact Form Validation

### PR DESCRIPTION
**Pull request by adamilare**
In this exercise I add the following to the portfolio project;
- Contact form;
- Validation of the contact form fields.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
...
+              id="form-contact"
+              method="POST"
+              action="https://formspree.io/f/xlekdbpd"
+            >
+              <input
+                class="form-text"
+                type="text"
+                name="name"
+                placeholder="Full name"
+                maxlength="50"
```
Comment (pinkmoon25) [inline index.html]: - [ ] good job implemting the contact form :clap:, however please notice that as mentioned in the project requirements we have to make sure Name field accepts up to 30 chars, I would suggest to update the maxlength attribute to 30.

File: index.html [hunk 2]
```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
...
                 type="email"
+                name="email"
                 placeholder="Email address"
+                required
               />
+
               <textarea
                 class="form-textarea"
-                name=""
                 id=""
```
Comment (pinkmoon25) [inline index.html]: - [ ] Please remove the emty attributes, it is not in the HTML CSS best practices to commit an empty code to the codebase, Please remove this id attribute if you do not need it

File: index.html [hunk 3]
```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
             </p>
```
Comment (pinkmoon25) [inline index.html:180]: - [ ] Please notice that in the original figma design the font-size property for this paragraph is 32px, you have set it to 28px currently, as one of the main goal of the project is to parse a figma template as close to as possible, please fix the font-size propert
currently: 
```
#contact p {
  font-size: 28px;
}
```

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @adamilare , 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
Form Implementation :heavy_check_mark: 
HTML validation :heavy_check_mark: 
Integrate formspree :heavy_check_mark: 
Used flex :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (adamilare) [reply index.html:180]: @PinkMoon25 Thank you for pointing those out

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Formspree service integrated successfully :heavy_check_mark: 
- Good UI design :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (pinkmoon25) [reply index.html:180]: @adamilare you got it 👍

**Pull request merged**